statviz/src/views/Dashboard/Dashboard.tsx

```diff
@@ -44,10 +31,11 @@ export default function Dashboard() {
         </WrapItem>
```

Comment:
FYI: changes to this TimeRangeSelect will cause a re-render of the Dashboard component  and subsequently cause a query to be sent.